test/images/sample-apiserver/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE AS build_k8s_1_7_sample_apiserver
+
+ENV GOPATH /go
+ENV GOBIN /usr/lib/go-1.9/bin
+ENV PATH ${GOBIN}:$PATH
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends software-properties-common
+
+RUN add-apt-repository ppa:gophers/archive && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends build-essential ca-certificates golang-1.9-go git
```

Comment:
Why not install go the official/real way as done in the other build dockerfiles?

Reply:
We need to build the sample-apiserver from the 1.7-release branch. I don't see any other way to do this. Any suggestions?

Reply:
See https://github.com/docker-library/golang/blob/master/1.9/stretch/Dockerfile for reference how to do it

Reply:
Ack fixing